modules/programs/khal.nix

```
@@ -158,6 +157,7 @@ in {
           # locale = definedAttrs (cfg.locale // { _module = null; });
 
           default = optionalAttrs (!isNull primaryAccount) {
+            highlight_event_days = True;
```

```suggestion
            highlight_event_days = true;
```

Could you add a test so that this would have been caught?

---

I will take a look at writing tests.